nixos/tests/all-tests.nix

```diff
@@ -754,6 +754,7 @@ in
   lemmy = runTest ./lemmy.nix;
   libinput = runTest ./libinput.nix;
   librenms = runTest ./librenms.nix;
+  librenms-agent = handleTest ./librenms-agent.nix { };
```

Comment:
Could you convert the test to use the new `runTest`? It should only require dropping `import ./make-test-python.nix` from the test